tests/rptest/services/redpanda.py

```diff
@@ -2214,6 +2153,22 @@ def cluster_unhealthy_reason(self) -> str | None:
     def cluster_healthy(self) -> bool:
         return self.cluster_unhealthy_reason is not None
 
+    def raise_on_bad_logs(self, allow_list=None):
+        """
+        Raise a BadLogLines exception if any nodes' logs contain errors
+        not permitted by `allow_list`
+
+        :param logger: the test's logger, so that reports of bad lines are
```

Comment:
Looks like logger is not being used here. might want to remove from these docs if not going to use it

Reply:
Yes, will do.